src/main/scala/dotty/semanticdb/Utils.scala

```diff
@@ -28,7 +28,7 @@ object Utils {
   /** List all tasty files occuring in the folder f or one of its subfolders */
   def recursiveListFiles(f: File, prefix : String = ""): Array[File] = {
     val pattern = (".*" + prefix + ".*\\.tasty").r
-    val files = f.listFiles
+    val files = f.listFiles.map(_.nn)
```

Comment:
A Java method/field that would need a more precise type annotation than just null/non-null at the outermost level.